MovieJournalMVC/Controller/MasterViewController.swift

```diff
@@ -9,73 +9,82 @@
 import UIKit
 
 class MasterViewController: UITableViewController {
-
+    
     var detailViewController: DetailViewController? = nil
-    var objects = [Any]()
-
-
+    var objects = Store.shared.movieReviews
```

Comment:
It would be a good idea to more properly naming your variables. Perhaps `reviews`?

Reply:
I renamed variables and classes to more sensible names!